Source/WebCore/crypto/mac/CryptoAlgorithmAES_GCMMac.cpp

```diff
@@ -69,8 +78,11 @@ ALLOW_DEPRECATED_DECLARATIONS_END
     return WTFMove(plainText);
 }
 
-ExceptionOr<Vector<uint8_t>> CryptoAlgorithmAES_GCM::platformEncrypt(const CryptoAlgorithmAesGcmParams& parameters, const CryptoKeyAES& key, const Vector<uint8_t>& plainText)
+ExceptionOr<Vector<uint8_t>> CryptoAlgorithmAES_GCM::platformEncrypt(const CryptoAlgorithmAesGcmParams& parameters, const CryptoKeyAES& key, const Vector<uint8_t>& plainText, bool useCryptoKit)
```

Comment:
Instead of adding a bool parameter, we may want to add a new CryptoAlgorithmAES_GCM that only uses CryptoKit.  It's nbd to add a bool in one place, but as we expand this it may be structured better to just have a new implementation object.

Reply:
As discussed:
1. bool will still be required to select between the two at runtime.
2. Once I make CK the default this bool will just go away.